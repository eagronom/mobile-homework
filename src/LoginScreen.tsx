import React, { Fragment } from 'react';
import { Image } from 'react-native';

class LoginScreen extends React.PureComponent {
  render = () => (
    <Fragment>
      <Image source={require('../assets/eagronom.png')} />
    </Fragment>
  )
}

export default LoginScreen;
