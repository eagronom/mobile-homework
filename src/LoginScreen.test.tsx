import React from 'react';
import { shallow } from 'enzyme';
import LoginScreen from './LoginScreen';

describe('<LoginScreen />', () => {
  given('component', () => shallow(<LoginScreen {...given.props} />));
  given('props', () => ({}));

  it('renders', () => {
    expect(given.component).toMatchSnapshot();
  });
});
