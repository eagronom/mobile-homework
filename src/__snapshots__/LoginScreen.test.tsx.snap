// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginScreen /> renders 1`] = `
<Fragment>
  <Image
    source={
      Object {
        "testUri": "../../../assets/eagronom.png",
      }
    }
  />
</Fragment>
`;
